### PR TITLE
Color Schemes: Remove Laser Black

### PR DESCRIPTION
**Pull request by sixhours**
#### Changes proposed in this Pull Request

* No more Laser Black.

#### Testing instructions

* Switch to this PR and go to `/me/account/` on calypso.live. Laser Black should not appear in the list of color schemes.

### CODE REVIEW

**Review by blowery — APPROVED** [review]
LGTM

We could drop the values from color-schemes too

**Comment by sixhours** [comment]
> We could drop the values from color-schemes too

I wondered about removing it entirely in case we wanted to reference it/use it for testing color contrast bugs again. Although I suppose we do have version control. :)

**Comment by blowery** [comment]
> I wondered about removing it entirely in case we wanted to reference it/use it for testing color contrast bugs again.

It would be nice to slim down that file a touch if we can. I'd rather drop it just to reduce possible confusion. 

> Although I suppose we do have version control. :)

And that! :)

File: client/blocks/color-scheme-picker/style.scss [hunk 1]
```diff
@@ -19,7 +19,4 @@
 		background-image: url( '/calypso/images/color-schemes/color-scheme-thumbnail-sakura.svg' );
 	}
 
-	&.is-laser-black {
-		background-image: url( '/calypso/images/color-schemes/color-scheme-thumbnail-dark.svg' );
```
Comment (torres126) [inline client/blocks/color-scheme-picker/style.scss]: Can't the image itself also be removed now?

**Comment by tyxla** [comment]
Hey folks - is it the case that we no longer have dark background color schemes? We introduced a conditional for this in one of the devdocs examples: https://github.com/Automattic/wp-calypso/blob/b88c6a655cc78c6b25c19053b47da2f547738715/client/components/jetpack-header/docs/example.js#L29 and it seems we can clean that up now. Let me know if that's the case.

**Comment by sixhours** [comment]
@tyxla Yes indeed!

**Comment by tyxla** [comment]
Thanks @sixhours! Cleaning it up in #32099.

**Pull request merged**